diff --git a/node_modules/@matt-block/react-native-in-app-browser/android/src/main/kotlin/com/mattblock/reactnative/inappbrowser/RNInAppBrowserModule.kt b/node_modules/@matt-block/react-native-in-app-browser/android/src/main/kotlin/com/mattblock/reactnative/inappbrowser/RNInAppBrowserModule.kt
index b906b90..09a2c45 100644
--- a/node_modules/@matt-block/react-native-in-app-browser/android/src/main/kotlin/com/mattblock/reactnative/inappbrowser/RNInAppBrowserModule.kt
+++ b/node_modules/@matt-block/react-native-in-app-browser/android/src/main/kotlin/com/mattblock/reactnative/inappbrowser/RNInAppBrowserModule.kt
@@ -14,7 +14,7 @@ import android.graphics.BitmapFactory
 import android.graphics.Color
 import android.net.Uri
 import android.os.Bundle
-import android.support.customtabs.*
+import androidx.browser.customtabs.*
 
 import com.facebook.react.bridge.*
 import com.facebook.react.uimanager.PixelUtil
