diff --git a/node_modules/@react-native-community/cli/build/commands/runIOS/findMatchingSimulator.js b/node_modules/@react-native-community/cli/build/commands/runIOS/findMatchingSimulator.js
index 4f75b7a..081fd38 100644
--- a/node_modules/@react-native-community/cli/build/commands/runIOS/findMatchingSimulator.js
+++ b/node_modules/@react-native-community/cli/build/commands/runIOS/findMatchingSimulator.js
@@ -65,7 +65,7 @@ function findMatchingSimulator(simulators, simulatorString) {
     for (const i in device) {
       const simulator = device[i]; // Skipping non-available simulator

-      if (simulator.availability !== '(available)' && simulator.isAvailable !== 'YES') {
+      if (simulator.availability !== '(available)' && simulator.isAvailable !== true) {
         continue;
       }

