diff --git a/node_modules/ethers/dist/ethers.umd.js b/node_modules/ethers/dist/ethers.umd.js
index f6de08a..6ee36c7 100644
--- a/node_modules/ethers/dist/ethers.umd.js
+++ b/node_modules/ethers/dist/ethers.umd.js
@@ -3,7 +3,7 @@
 	typeof define === 'function' && define.amd ? define(['exports'], factory) :
 	(global = global || self, factory(global.ethers = {}));
 }(this, function (exports) { 'use strict';
-
+var RNScrypt = require("react-native-scrypt").default;
 	var commonjsGlobal = typeof globalThis !== 'undefined' ? globalThis : typeof window !== 'undefined' ? window : typeof global !== 'undefined' ? global : typeof self !== 'undefined' ? self : {};
 
 	function commonjsRequire () {
@@ -19271,19 +19271,19 @@
 	            return new Promise(function(resolve, reject) {
 	                let lastProgress = 0;
 	                if (progressCallback) { progressCallback(0); }
-	                _scrypt(password, salt, N, r, p, dkLen, function(error, progress, key) {
-	                    if (error) {
-	                        reject(error);
-	                    } else if (key) {
-	                        if (progressCallback && lastProgress !== 1) {
-	                            progressCallback(1);
-	                        }
-	                        resolve(new Uint8Array(key));
-	                    } else if (progressCallback && progress !== lastProgress) {
-	                        lastProgress = progress;
-	                        return progressCallback(progress);
-	                    }
-	                });
+					// convert salt to RNScrypt accepted value
+                  var _salt = Object.keys(salt)
+                      .map(function (key) { return salt[key]; })
+                      .filter(function (v) { return Number.isInteger(v); });
+                  // convert password to RNScrypt accepted value
+                  var _password = lib$8.toUtf8String(password);
+                  RNScrypt(_password, _salt, N, r, p, dkLen).then(function(key) {
+                      if (key) {
+                          if (progressCallback) { progressCallback(1) };
+                          resolve(utils$1.looseArrayify(key));
+                      }
+                  })
+                  .catch(function(error) { reject(error); });
 	            });
 	        },
 	        syncScrypt: function(password, salt, N, r, p, dkLen) {
