diff --git a/node_modules/lottie-ios/lottie-ios.podspec b/node_modules/lottie-ios/lottie-ios.podspec
index 970e41d..3c4876f 100644
--- a/node_modules/lottie-ios/lottie-ios.podspec
+++ b/node_modules/lottie-ios/lottie-ios.podspec
@@ -22,7 +22,7 @@ For the first time, designers can create and ship beautiful animations without a
   s.author           = { 'Brandon Withrow' => 'buba447@gmail.com' }
   s.source           = { :git => 'https://github.com/airbnb/lottie-ios.git', :tag => s.version.to_s }
 
-  s.swift_version = '5.0'
+  s.swift_version = '4.2'
   s.ios.deployment_target = '9.0'
   s.osx.deployment_target = '10.10'
   s.tvos.deployment_target = '9.0'
