diff --git a/node_modules/pouchdb-adapter-asyncstorage/src/asyncstorage_core.js b/node_modules/pouchdb-adapter-asyncstorage/src/asyncstorage_core.js
index a4c817e..3dc908a 100644
--- a/node_modules/pouchdb-adapter-asyncstorage/src/asyncstorage_core.js
+++ b/node_modules/pouchdb-adapter-asyncstorage/src/asyncstorage_core.js
@@ -4,7 +4,7 @@
  * Adapted from https://github.com/tradle/asyncstorage-down
  */
 
-import { AsyncStorage } from 'react-native'
+import AsyncStorage from '@react-native-async-storage/async-storage';
 import {
   safeJsonParse,
   safeJsonStringify
diff --git a/node_modules/pouchdb-adapter-asyncstorage/src/index.js b/node_modules/pouchdb-adapter-asyncstorage/src/index.js
index 39bad82..b1041c8 100644
--- a/node_modules/pouchdb-adapter-asyncstorage/src/index.js
+++ b/node_modules/pouchdb-adapter-asyncstorage/src/index.js
@@ -109,7 +109,7 @@ function AsyncStoragePouch (dbOpts, constuctorCallback) {
 
 AsyncStoragePouch.valid = () => {
   try {
-    return require('react-native').AsyncStorage !== null
+    return require('@react-native-async-storage/async-storage').default !== null
   } catch (error) {
     return false
   }
