diff --git a/node_modules/react-native-svg-animations/utils/AnimatedSVGBrush/index.js b/node_modules/react-native-svg-animations/utils/AnimatedSVGBrush/index.js
index 75ebdef..e2f2c82 100644
--- a/node_modules/react-native-svg-animations/utils/AnimatedSVGBrush/index.js
+++ b/node_modules/react-native-svg-animations/utils/AnimatedSVGBrush/index.js
@@ -114,7 +114,7 @@ export default function SvgBrushFix(WrappedComponent) {
       }
       this._component && this._component.setNativeProps(props);
     };
-    componentWillReceiveProps(nextProps) {
+    UNSAFE_componentWillReceiveProps(nextProps) {
       this.updateCache(nextProps);
       if (nextProps.strokeDasharray !== this.props.strokeDasharray) {
         removeListeners(this.strokeDasharray);
