diff --git a/node_modules/react-native-sentry/lib/raven-plugin.js b/node_modules/react-native-sentry/lib/raven-plugin.js
index a4ca01d..73bad96 100644
--- a/node_modules/react-native-sentry/lib/raven-plugin.js
+++ b/node_modules/react-native-sentry/lib/raven-plugin.js
@@ -176,7 +176,7 @@ function reactNativePlugin(Raven, options, internalDataCallback) {
  * Returns a promise that guarantees never to reject.
  */
 reactNativePlugin._persistPayload = function(payload) {
-  var AsyncStorage = require('react-native').AsyncStorage;
+  var AsyncStorage = require('@react-native-community/async-storage').default;
   return AsyncStorage.setItem(ASYNC_STORAGE_KEY, JSON.stringify(payload))[
     'catch'
   ](function() {
@@ -190,7 +190,7 @@ reactNativePlugin._persistPayload = function(payload) {
  * Returns a promise that guarantees never to reject.
  */
 reactNativePlugin._restorePayload = function() {
-  var AsyncStorage = require('react-native').AsyncStorage;
+  var AsyncStorage = require('@react-native-community/async-storage').default;
   var promise = AsyncStorage.getItem(ASYNC_STORAGE_KEY).then(function(payload) {
     return JSON.parse(payload);
   })['catch'](function() {
@@ -213,7 +213,7 @@ reactNativePlugin._restorePayload = function() {
  * Clears any persisted payloads.
  */
 reactNativePlugin._clearPayload = function() {
-  var AsyncStorage = require('react-native').AsyncStorage;
+  var AsyncStorage = require('@react-native-community/async-storage').default;
   return AsyncStorage.removeItem(ASYNC_STORAGE_KEY)['catch'](function() {
     return null;
   });
