diff --git a/node_modules/react-native-appsflyer/react-native.config.js b/node_modules/react-native-appsflyer/react-native.config.js
index dfbe81b..6a653a1 100644
--- a/node_modules/react-native-appsflyer/react-native.config.js
+++ b/node_modules/react-native-appsflyer/react-native.config.js
@@ -3,7 +3,6 @@ const path = require('path');
 module.exports = {
   dependency: {
     platforms: {
-      ios: { podspecPath: path.join(__dirname, 'react-native-appsflyer.podspec') },
       android: {
         packageImportPath: 'import com.appsflyer.reactnative.RNAppsFlyerPackage;',
         packageInstance: 'new RNAppsFlyerPackage()',
