diff --git a/node_modules/react-native-svg/src/css.tsx b/node_modules/react-native-svg/src/css.tsx
index f567b80..eb75078 100644
--- a/node_modules/react-native-svg/src/css.tsx
+++ b/node_modules/react-native-svg/src/css.tsx
@@ -695,14 +695,22 @@ export function SvgCssUri(props: UriProps) {
   const { uri, onError = err, onLoad } = props;
   const [xml, setXml] = useState<string | null>(null);
   useEffect(() => {
-    uri
-      ? fetchText(uri)
+    const fetchController = new AbortController();
+    let mounted = true;
+    (() => {
+      if(!uri) return
+      fetchText(uri)
           .then((data) => {
+            if(mounted){
             setXml(data);
             onLoad?.();
+            }
           })
           .catch(onError)
-      : setXml(null);
+    })()  
+    return () => {
+      mounted = false;
+      fetchController.abort()};
   }, [onError, uri, onLoad]);
   return <SvgCss xml={xml} override={props} />;
 }
diff --git a/node_modules/react-native-svg/src/lib/extract/extractText.tsx b/node_modules/react-native-svg/src/lib/extract/extractText.tsx
index 96d9be2..75f1e9b 100644
--- a/node_modules/react-native-svg/src/lib/extract/extractText.tsx
+++ b/node_modules/react-native-svg/src/lib/extract/extractText.tsx
@@ -24,12 +24,8 @@ function extractSingleFontFamily(fontFamilyString?: string) {
   // SVG on the web allows for multiple font-families to be specified.
   // For compatibility, we extract the first font-family, hoping
   // we'll get a match.
-  return fontFamilyString
-    ? fontFamilyString
-        .split(commaReg)[0]
-        .replace(fontFamilyPrefix, '')
-        .replace(fontFamilySuffix, '')
-    : null;
+  // custom fonts will crash the app so we're using just the default font
+  return null;
 }
 
 function parseFontString(font: string) {
