diff --git a/node_modules/react-native-cached-image/CachedImage.js b/node_modules/react-native-cached-image/CachedImage.js
index daf03c8..611b3b8 100644
--- a/node_modules/react-native-cached-image/CachedImage.js
+++ b/node_modules/react-native-cached-image/CachedImage.js
@@ -3,6 +3,7 @@
 const _ = require('lodash');
 const React = require('react');
 const ReactNative = require('react-native');
+const NetInfo = require('@react-native-community/netinfo');
 
 const PropTypes = require('prop-types');
 
@@ -16,7 +17,6 @@ const {
     View,
     ImageBackground,
     ActivityIndicator,
-    NetInfo,
     Platform,
     StyleSheet,
 } = ReactNative;
@@ -79,12 +79,12 @@ class CachedImage extends React.Component {
 
     componentWillMount() {
         this._isMounted = true;
-        NetInfo.isConnected.addEventListener('connectionChange', this.handleConnectivityChange);
+        this.removeNetInfoEventListener = NetInfo.addEventListener(this.handleConnectivityChange);
         // initial
-        NetInfo.isConnected.fetch()
-            .then(isConnected => {
+        NetInfo.fetch()
+            .then((netInfoState) => {
                 this.safeSetState({
-                    networkAvailable: isConnected
+                    networkAvailable: netInfoState.isInternetReachable
                 });
             });
 
@@ -93,7 +93,10 @@ class CachedImage extends React.Component {
 
     componentWillUnmount() {
         this._isMounted = false;
-        NetInfo.isConnected.removeEventListener('connectionChange', this.handleConnectivityChange);
+        if (this.removeNetInfoEventListener) {
+            this.removeNetInfoEventListener();
+            delete this.removeNetInfoEventListener;
+        }
     }
 
     componentWillReceiveProps(nextProps) {
